src/main/groovy/com/twcable/grabbit/jcr/JCRNodeDecorator.groovy

```diff
@@ -105,6 +106,26 @@ class JCRNodeDecorator {
         return definition.isMandatory()
     }
 
+    /**
+     * Returns the "jcr:created", "jcr:lastModified" or "cq:lastModified" date property
+     * for current Jcr Node
+     * Returns null of none of the 3 are found
+     * @return
+     */
+    public Date getDate() {
+        final String CQ_LAST_MODIFIED = "cq:lastModified"
```

Comment:
Yeah, I could go either way. FWIW, we already transitively use com.day.cq.wcm:cq-wcm-api:5.6.2